lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.html

```diff
@@ -2,26 +2,24 @@
     <div
         [attr.data-automation-id]="'card-select-label-' + property.key"
         class="adf-property-label"
+        [ngClass]="{'adf-property-value-editable' : isEditable()}"
```

Comment:
we should avoid calling functions in html

Reply:
Done